configs/compliance.json

```diff
@@ -4,7 +4,7 @@
       "name": "Compliance Access",
       "description": "An all access role which grants read and write permissions.",
       "system": true,
-      "version": 2,
+      "version": 3,
```

Comment:
Same with compliance, I don't think it needs a version bump. That's only for the app to pickup/apply changes.